functions-methods.js

```diff
@@ -9,8 +10,16 @@
 // getEmailDomain("t.mellink@novi.nl") geeft novi.nl
 // getEmailDomain("a.wiersma@outlook.com") geeft outlook.com
 
+function getEmailDomain (emailAddress) {
+    let indexAt = emailAddress.indexOf("@");
```

Comment:
kijk nog eens naar deze functie, het eerste gedeelte is bijna goed. Als je gebruik maakt van methode SPLIT en indexAt returned[1] vanaf index nummer 1 dan heb je een kortere leesbaardere code